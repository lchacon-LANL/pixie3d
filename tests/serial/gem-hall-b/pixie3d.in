 &datin
    ilevel   = 0
    test     = t

    nxd      = 64
    nyd      = 64 
    nzd      = 1

    numerical_grid = f

    xmax     = 6.4
    ymax     = 12.8
    zmax     = 1d-3
    bcs      = 'def','sym','sym','sym','per','per'

    eta      = 5d-3
    nu       = 5d-2
    dd       = 0d-3
    chi      = 2d-2
    heta     = 1d-4

    di       = 1d0
    de       = 0.

    temp_ratio = 5.

    equil    = 'gem'
    coords   = 'car'
    gp1      = .true.,6.4d0,1d-2

    dlambda  = .5
    rshear   = 1.
    vperflow = 0.

    dt       = 1d-2
    tmax     = 1d-1
    dstep    = 1d-1

    pbx      = 0.1
    pby      = 0.1
    pbz      = 0.
    nh2      = 1

    cnfactor = 0.5
    sm_flag  = 0

    restart  = f
    timecorr = t
    predictor= f

    source   = f

    precon   = 'id'
    mgtol    = 1d-3
    maxvcyc  = 10
    nsweep   = 4
    precpass = 0

    iguess   = 0

    maxksp   = 20
    maxitgm  = 20
    maxitnwt = 20
    tolgm    = 8d-1
    rtol     = 1d-4
    method   = 1
    mf_eps   = 1d-6

    solenoidal = t
    nc_eom_f   = f
    nc_eom_v   = f
    adiabatic  = f
    ion_hall   = f
    fake_ve    = f
    sym_st     = t
    advect     = 4
    slava      = t
 
    debug    = f
    debug_it = 5

    check_grid = f
    chk_src  = f
 /

 &graphdef
    sel_diag  = 2,9,10,11,12,13,15,16,17,18,19,20,28,29
    sel_graph = 1,2,3,4,5,6,7,8,9
    dplot = 1d-4
    prof_conf = 1,'x',1,1,1
    cont_conf = 3,'x','y',1,1,1
    iplot = 64
    jplot = 1 
    kplot = 1
    hdf_plot = f
    xdraw_plot = t
    adios_plot = f
 /
