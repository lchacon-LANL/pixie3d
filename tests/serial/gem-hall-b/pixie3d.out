
  Grid mesh:................   64x   64x    1
  Number of grid levels:....    5,    5,    1
  Preconditioning method:...  id
  Time correction...........   T
  External source...........   F
  Numerical grid............   T
  Advection scheme..........  SMART   
  TS scheme................. Theta    

   itime    time     dt     vx_max   vy_max   vz_max  GMRES  Newton  CN factor  GMRES-DC
      1    0.000  4.8E-04   2.7E-05  7.2E-06  2.6E-09     2     2     0.500        0
      2    0.010  1.0E-02   5.8E-04  1.6E-04  1.2E-06     9     5     0.500        0
      3    0.020  1.0E-02   1.1E-03  3.0E-04  4.7E-06     8     5     0.500        0
      4    0.030  1.0E-02   1.7E-03  4.5E-04  1.0E-05    11     5     0.500        0
      5    0.040  1.0E-02   2.2E-03  6.0E-04  1.8E-05     9     5     0.500        0
      6    0.050  1.0E-02   2.8E-03  7.5E-04  2.9E-05    12     5     0.500        0
      7    0.060  1.0E-02   3.3E-03  9.0E-04  4.1E-05    10     5     0.500        0
      8    0.070  1.0E-02   3.9E-03  1.0E-03  5.6E-05     9     5     0.500        0
      9    0.080  1.0E-02   4.4E-03  1.2E-03  7.3E-05    12     5     0.500        0
     10    0.090  1.0E-02   5.0E-03  1.3E-03  9.2E-05    10     5     0.500        0
     11    0.100  1.0E-02   5.5E-03  1.5E-03  1.1E-04    11     5     0.500        0

Final statistics

 itime  Newt/ndt  GMRES/ndt  GMRES/Newt
   11       4.7        9.4        2.0
